app/views/layouts/application.html.erb

```diff
@@ -18,8 +18,8 @@
   <%= javascript_include_tag "https://cdnjs.cloudflare.com/ajax/libs/jquery/2.2.4/jquery.min.js" %>
   <%= javascript_include_tag "https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js" %>
   <%= javascript_include_tag "https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.18.1/moment.min.js" %>
-  <%= javascript_pack_tag 'vendor', 'data-turbolinks-track': 'reload' %>
-  <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
+  <!-- %= javascript_pack_tag 'vendor', 'data-turbolinks-track': 'reload' % >
+  < %= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' % -->
```

Comment:
This makes the JS not load.

Reply:
Fixed, thanks

Reply:
Oh god, thanks for catching this. I disabled it because webpack is broken in my dev env, and it wouldn't load until it got a timeout.